src/hatti/views/table.cljs

```diff
@@ -303,6 +303,21 @@
       (om/set-state! owner :dataview dataview)
       [grid dataview])))
 
+(defn get-table-view-height
+  []
+  (let [body-height (-> "body"
+                        js/document.querySelector
+                        .-clientHeight)
+        main-navigation-height (-> "#dataview-menu"
+                                   js/document.querySelector
+                                   .-clientHeight)
+        tab-navigation-height (-> ".tab-bar"
+                                  js/document.querySelector
+                                  .-clientHeight)]
+    (- body-height
+       (+ main-navigation-height
+          tab-navigation-height))))
+
```

Comment:
Still wondering how these got overwritten on this branch. I'd done this here: f150339fcd4a9d4052c01dfac9e6863baa85162d.